riff-raff/app/controllers/Application.scala

```diff
@@ -84,19 +86,19 @@ class Application(prismLookup: PrismLookup, deploymentTypes: Seq[DeploymentType]
     Ok(views.html.index(request, documentation))
   }
 
-  def deployInfoData = AuthAction { implicit request =>
+  def deployInfoData = authAction { implicit request =>
```

Comment:
Would this perhaps make a little more sense if we were to pass round an `auth` instance which is invoked with `auth.Action`?